### PR TITLE
compress: log compression ratio

### LINKED ISSUE #423: compress: log original file size or compression ratio
It's hard to know how well the compression is performing without seeing the compression ratio achieved.

### PR DESCRIPTION
**Pull request by mmilata**
Closes #423.
Or do we want the uncompressed size in metadata as well?

### CODE REVIEW

File: atomic_reactor/plugins/post_compress.py [hunk 1]
```diff
@@ -85,5 +88,12 @@ def run(self):
...
                 outfile = self._compress_image_stream(image_stream)
-        self.workflow.exported_image_sequence.append(get_exported_image_metadata(outfile))
+        metadata = get_exported_image_metadata(outfile)
+        self.workflow.exported_image_sequence.append(metadata)
+
+        compressed_size = metadata['size']
+        if compressed_size != 0:
+            self.log.debug('uncompressed: %s, compressed: %s, ratio: %.2f',
+                human_size(self.uncompressed_size), human_size(compressed_size),
+                self.uncompressed_size/float(compressed_size))
```
Comment (twaugh) [inline atomic_reactor/plugins/post_compress.py]: Nice! Does this ratio calculation agree with what `gzip -l` says though? I think it should be size_decrease / uncompressed_size.

**Comment by twaugh** [comment]
It would be useful to also have that information in the metadata, yes: both compressed and uncompressed size. There might be some small overlap if/when we include full koji metadata in the OSv3 metadata.

File: atomic_reactor/util.py [hunk 2]
```diff
@@ -567,3 +567,12 @@ def get_build_json():
     except KeyError:
         logger.error("No $BUILD env variable. Probably not running in build container")
         raise
+
+# copypasted and slightly modified from
+# http://stackoverflow.com/questions/1094841/reusable-library-to-get-human-readable-version-of-file-size/1094933#1094933
+def human_size(num, suffix='B'):
```
Comment (TomasTomecek) [inline atomic_reactor/util.py]: don't want to pick the nits so hard but this could use some tests too :)

**Comment by mmilata** [comment]
Sorry, I blindly used the first formula from [wikipedia](https://en.wikipedia.org/wiki/Data_compression_ratio). It looks like compression tools use the second variant (which is more intuitive anyway). Updated the branch to use the other formula, store uncompressed size in metadata, and test the `human_size` function as requested.

**Comment by twaugh** [comment]
LGTM! Merging.

**Pull request merged**